texar/tf/modules/decoders/rnn_decoder_helpers.py

```diff
@@ -381,18 +381,14 @@ def sample(self, time, outputs, state, name=None):
         sample_ids = tf.nn.softmax(outputs / self._tau)
         return sample_ids
 
-    def next_inputs(self, time, outputs, state, sample_ids, name=None,
-                    reach_max_time=None):
+    def next_inputs(self, time, outputs, state, sample_ids, name=None):
```

Comment:
`tf_helpers.py` (https://github.com/asyml/texar/blob/4c8b0c0dc2f03d8557ed66a7727fa7efa77cfa08/texar/tf/modules/decoders/tf_helpers.py) also need updates -- delete `reach_max_time` and probably also `unused_kwargs`